jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -67,44 +58,29 @@ def Event_Status = [
 
 @Field def created_by = ''
 
-@Field def role = ''
 @Field def current_environment = ''
-@Field def region = ''
 @Field def jq = '/usr/local/bin/jq'
 
 node ()  {
-	jazzBuildModuleURL = "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"
+	def jazzBuildModuleURL = "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"
 	loadConfigModule(jazzBuildModuleURL)
-	var_api_key 				= configLoader.API.API_KEY
-	var_api_id_dev = configLoader.API.API_ID_DEV
-	var_aws_credential_id = configLoader.AWS_CREDENTIAL_ID
-	var_env_name_prefix = configLoader.env_name_prefix
-	var_repo_core = configLoader.REPOSITORY.REPO_CORE
-	var_repo_loc = configLoader.REPOSITORY.REPO_LOC
-	var_credentialsId 			= configLoader.REPOSITORY.REPO_CREDENTIAL_ID
-	var_bitbucket_base			= configLoader.REPOSITORY.REPO_BASE
-	var_bitbucket_email 		= configLoader.REPOSITORY.REPO_EMAIL
-	region = configLoader.JAZZ.jazz_region
-	g_base_url = "https://${configLoader.API.API_KEY}.execute-api.${region}.amazonaws.com/prod"
-	role = configLoader.JAZZ.jazz_roleId
-	api_doc_s3_bucket = configLoader.JAZZ_S3.jazz_s3_api_doc
-	def jazzBuildModuleURL = "http://${var_bitbucket_base}/scm/"+ var_repo_core +"/jazz-build-module.git"
-
-	echo "Jobs Params: " + params
-	echo "Build triggered via branch: " + params.scm_branch
-	def _event = ""
 
-    echo "start"
+	g_base_url = "https://${configLoader.API.API_KEY}.execute-api.${configLoader.JAZZ.jazz_region}.amazonaws.com/prod"
+	
+	echo "Build triggered via branch: " + params.scm_branch + " with params " + params
+	def _event = ""
 
     def serviceName = ""
     def cfStackName = ""
-		if(params.service_name != null && !params.service_name.isEmpty()){
-			serviceName = params.service_name
-			cfStackName = params.service_name
-		}
+	
+	if(params.service_name != null && !params.service_name.isEmpty()){
```

Comment:
groogy has a simpler way of checking null or empty. we can just use  if(params.service_name){..}

Reply:
Cool, will use it for next set of cleanup